src/FakeItEasy.Analyzer/SymbolHelpers.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace FakeItEasy.Analyzer
+{
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CSharp;
+    using Microsoft.CodeAnalysis.CSharp.Syntax;
+    using Microsoft.CodeAnalysis.Diagnostics;
+
+    internal static class SymbolHelpers
+    {
+        /// <summary>
+        /// Returns the called method symbol for the invocation expression
+        /// </summary>
+        /// <param name="call"></param>

```

Comment:
I tell people at work that empty param and return elements just clutter the code. Could you provide the documentation, please?